entries/m/myfritz.net.json

```
@@ -0,0 +1,20 @@
+{
+  "MyFRITZ!Net": {
+    "domain": "myfritz.net",
+    "url": "https://www.myfritz.net",
+    "additional-domains": [
+      "sso.myfritz.net"
+    ],
+    "tfa": [
+      "totp",
```

```suggestion
      "totp",
      "u2f",
```

Their documentation says they support U2F security keys